stylesheets/_component.search.scss

```diff
@@ -16,6 +16,13 @@
     }
 }
 
+// Set margin and position of loading spinner when directly adjacent to the search button
+.main-search .loading {
+    position: absolute;
+    right: -30px;
```

Comment:
How does this look on mobile when the search is 100% width?

Reply:
Option with spinner inside the search element?

![Screenshot 2019-12-03 at 11 11 27](https://user-images.githubusercontent.com/18653/70046320-b6e9cf80-15bd-11ea-9656-9d0af64d9257.png)

Reply:
![Screenshot 2019-12-03 at 11 53 50](https://user-images.githubusercontent.com/18653/70049535-cd475980-15c4-11ea-9b66-15f18061168b.png)

Positioned in sidebar at narrow viewports